Automattic/newspack-ios
# Issues/83 app icon and colors

Refs #83 

This PR introduces:
- A first pass at an app icon. 
- Our ColorStudio assets and accompanying accessor code.

Almost all of this PR is boiler plate code. Only a few actual files need a look.  These are:
- ColorStudio.swift
- UIColor+ColorStudio.swift
- UIColor+SemanticColors.swift
These were borrowed from WooCommerce and then a few minor adjustments were made for Newspack. Basically a few things were renamed or removed.

ColorStudio colors are not yet used in the UI.  This will happen in subsequent PRs.

To test:
Switch to this branch, build and run the app. 
Confirm that the app builds and doesn't go 💥 
Confirm that you see a snazzy new app icon where before there was a placeholder in Springboard.

@jleandroperez Could I trouble you for a very colorful review :D 

Example icon:
<img width="85" alt="Screen Shot 2020-07-15 at 2 59 28 PM" src="https://user-images.githubusercontent.com/1435271/87590081-d72d5100-c6ab-11ea-85b6-f25f3373a2d6.png">
